### PR TITLE
Feature/asi 236 size selection

### PR DESCRIPTION
**Pull request by Mark-xc**

### CODE REVIEW

**Comment by Mark-xc** [comment]
test

**Comment by XCPrabhu** [comment]
prabhu test comment

**Comment by rebeccaxc** [comment]
comment

File: cartridges/app_storefront_allheart/cartridge/client/default/js/product/base.js [hunk 1]
```diff
@@ -150,7 +157,12 @@ function processNonSwatchValues(attr, $productContainer) {
     var optionSelect;
 
     function showPrice(price) {
-        return (price && price != '$0.00' ? price : '');
+        if(price === '$0.00' || price == undefined) {
```
[the remark this answers is not in the document]
Comment (Mark-xc) [reply cartridges/app_storefront_allheart/cartridge/client/default/js/product/base.js]: I've updated the showprice function.

**Pull request merged**